README.md

```diff
@@ -208,6 +211,36 @@ $ hkt <command> <option>
 
 </details>
 
+<details><summary>Produce and consume Avro formatted messages</summary>
+
+Provided this schema for this `subject` `actors-value` in Schema Registry `http://localhost:8081`:
+
+```json
+{"type": "record", "name": "Actor", "fields": [{"type": "string", "name": "FirstName"}]}
+```
+
+It may produce the Avro formatted message:
```

Comment:
By using `may` do you mean it's not deterministic ? :p

Reply:
if the schema registry and/or kafka is up :-P